js/index.js

```diff
@@ -1 +1,84 @@
 // Your code goes here
+var btn = document.querySelectorAll('.btn');
+var modal = document.querySelector(".modal");
+var modalText = document.querySelector('.modal-content p')
+
+
+
+btn.forEach(btn => btn.addEventListener('mousedown', event => {
+    modal.style.display = 'block';
+   
+}));
+
+var span = document.getElementsByClassName("close")[0];
+
+
+
+span.onclick = function() {
+  modal.style.display = "none";
+}
+
+
+window.onclick = function(event) {
+  if (event.target == modal) {
+    modal.style.display = "none";
+  }
+}
```

Comment:
not sure if this code works.  if the user clicks on the modal, the modal doesn't disappear..